.github/workflows/ci.yaml

```diff
@@ -254,7 +254,7 @@ jobs:
 
     strategy:
       matrix:
-        k8s: [v1.26.15, v1.27.16, v1.28.15, v1.29.10, v1.30.6]
+        k8s: [v1.26.15, v1.27.16, v1.28.15, v1.29.12, v1.30.8]
```

Comment:
1.26/27 we hosuld deprecate and add 1.31/32

Reply:
This is for v1.11 & v1.10. 🙂